components/automate-cli/pkg/verifyserver/handlers/status/status.go

```diff
@@ -0,0 +1,40 @@
+package status
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/logger"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+type Status struct {
+	Logger logger.ILogger
+}
+
+type IStatus interface {
```

Comment:
Not very clear what is the need of the interface here. Since we are creating handler for all the endpoints which are unique by the request body